packages/nextjs/pages/index.tsx

```diff
@@ -394,15 +438,16 @@ const Home: NextPage<{
 };
 
 export const getStaticProps: GetStaticProps<{ stats: Stats }> = async () => {
-  const res = await fetch(`${process.env.NEXT_PUBLIC_BG_BACKEND_API}/api/stats`);
-
-  if (!res.ok) throw new Error(`Failed to fetch stats, received status ${res.status}`);
-
-  const stats = (await res.json()) as Stats;
-
   return {
     props: {
-      stats,
```

Comment:
I noticed you removed this section, I am assuming it's related to the `.env` errors? We will want to put this back before merging

Reply:
Ah feck, yeah I always forget about that when I commit again. Putting it back!